WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1965,6 +1986,22 @@ private extension ProductFormViewController {
     }
 }
 
+
+// MARK: Bottom sheet helpers
+//
+private extension ProductFormViewController {
+    func buildBottomSheetPresenter() -> BottomSheetPresenter {
+        BottomSheetPresenter(configure: { bottomSheet in
+            var sheet = bottomSheet
+            sheet.prefersEdgeAttachedInCompactHeight = true
+            sheet.largestUndimmedDetentIdentifier = .none
+            sheet.prefersGrabberVisible = true
+            sheet.detents = [.medium()]
```

Comment:
We should add `large` detent to make it easier to read when the font size is large and to be consistent with the bottom sheet on the campaign list.

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/11552/commits/22c4baf5308e6ffa91cd411907f1d0427131a918